hlapipeline/alignimages.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+
+"""
+This script is a modernized replacement of tweakreg.
+"""
+
+import argparse
+from astropy.io import fits
+from astropy.table import Table
+import pdb
+import sys
+from utils import astrometric_utils as amutils
+
+# Module-level dictionary contains instrument/detector-specific parameters used later on in the script.
+detector_specific_params = {"acs":
+                                {"hrc":
+                                     {"fwhmpsf":0.073, # TODO: Verify value
+                                      "platescale":0.025}, # TODO: hla DGSL pipleine value: 0.025. Instrument handbook value: 0.027, and later ~0.028 × 0.025"/pixel. Verify correct value.
+                                 "sbc":
+                                     {"fwhmpsf":0.065, # TODO: Verify value
+                                      "platescale":0.03}, # TODO: hla DGSL pipleine value: 0.030. Instrument handbook value: 0.032, and later ~0.034 × 0.030"/pixel Verify correct value.
+                                 "wfc":{"fwhmpsf":0.076, # TODO: Verify value
+                                      "platescale":0.05}}, # TODO: nothing. pipeline value in agreement with instrument handbook value.
+                            "wfc3":
+                                {"ir":{"fwhmpsf":0.14, # TODO: nothing. pipeline value in agreement with instrument handbook value.
+                                      "platescale":0.09}, # TODO: hla DGSL pipleine value: 0.09. Instrument handbook value: 0.13, also later 0.135×0.121. Verify correct value.
+                                 "uvis":{"fwhmpsf":0.076, # TODO: nothing. pipeline value in agreement with instrument handbook value.
+                                      "platescale":0.04}}} # TODO: hla DGSL pipleine value: 0.04. Instrument handbook value: 0.04, but also later 0.0395×0.0395. Verify correct value.
+#-----------------------------------------------------------------------------------------------------------------------
+def main(imgList):
+    """
+    Main calling function.
+
+    :param imgList: list of images
+    :type imgList: list
+    :return: nothing for now.
```

Comment:
Docstring needs to use numpydoc formatting as noted [in the style guide.](https://github.com/spacetelescope/style-guides/blob/master/guides/python.md).

Reply:
refomatted docstrings in commit [5258f1c](https://github.com/dulude/hlapipeline/commit/5258f1c054200f519e7ddcb07ac68f1fd408cc26)